spec/observables/dom/ajax-spec.ts

```diff
@@ -558,26 +555,17 @@ describe('ajax', () => {
       async: false,
     };
 
-    ajax(obj).subscribe(
-      (x: any) => {
```

Comment:
Nothing in here mattered, accept the check of `x.xhr.async`, but it's unremarkable that it's in the `subscribe` next callback. So I've cleaned this a bit to make the test simpler.